### ISSUE TITLE
wildbit/postmark-php 1.3.2 requires guzzlehttp/guzzle ~5.1 -> no matching package found.

### ISSUE THREAD

**Issue by btafoya** [wildbit/postmark-php#14]
While executing...

composer require wildbit/postmark-php

I get

```
  Problem 1
    - wildbit/postmark-php 1.3.2 requires guzzlehttp/guzzle ~5.1 -> no matching                                                                                                                                                                                                package found.
    - wildbit/postmark-php 1.3.1 requires guzzlehttp/guzzle ~5.1 -> no matching                                                                                                                                                                                                package found.
    - wildbit/postmark-php 1.3.0 requires guzzlehttp/guzzle ~5.1 -> no matching                                                                                                                                                                                                package found.
    - Installation request for wildbit/postmark-php ^1.3 -> satisfiable by wildb                                                                                                                                                                                               it/postmark-php[1.3.0, 1.3.1, 1.3.2].
```

My composer.json is:

```
{
    "require": {
		"guzzlehttp/guzzle": "^6.1",
		"laravel/homestead": "^2.1",
		"phpmailer/phpmailer": "dev-master",
		"linkorb/jsmin-php": "dev-master",
		"firephp/firephp-core": "dev-master",
		"mobiledetect/mobiledetectlib": "dev-master",
		"jv2222/ezsql": "dev-master",
		"drewm/mailchimp-api": "dev-master",
		"opauth/opauth": "*",
		"opauth/facebook": "*",
		"opauth/google": "*",
		"opauth/twitter": "*",
		"geoip2/geoip2": "~2.0",
		"twilio/sdk": "^4.2",
		"respect/validation": "^0.9.3",
		"egulias/email-validator": "~1.2"
    }
}

```

Yes I am utilizing "guzzlehttp/guzzle": "~6.0" which I am certain is the issue. How do I resolve this whichout dropping to version 5 of guzzle? Thanks!

**Comment by atheken** [comment]
@btafoya - This was an overdue update on our side. We had a branch called `updated-guzzle` which allowed you to use guzzle 6.0+ for awhile, but didn't merge it to master. We have merged it and released version `2.0` of the client. ([2.0.0 release notes](https://github.com/wildbit/postmark-php/releases/tag/2.0.0))

**Comment by btafoya** [comment]
Thank you! Your work is greatly appreciated!
